apps/addons/management/commands/sign_addons.py

```diff
@@ -27,12 +29,12 @@ def handle(self, *args, **options):
                 'Please provide at least one addon id to sign. If you want to '
                 'sign them all, use the "process_addons --task sign_addons" '
                 'management command.')
-        signing_server = options.get('signing_server', settings.SIGNING_SERVER)
-        preliminary_signing_server = options.get(
-            'preliminary_signing_server', settings.PRELIMINARY_SIGNING_SERVER)
+        full_server = options.get('signing_server') or settings.SIGNING_SERVER
+        prelim_server = (options.get('preliminary_signing_server') or
+                         settings.PRELIMINARY_SIGNING_SERVER)
 
         addon_ids = [int(addon_id) for addon_id in args]
         with override_settings(
-                SIGNING_SERVER=signing_server,
-                PRELIMINARY_SIGNING_SERVER=preliminary_signing_server):
-            sign_addons(addon_ids)
+                SIGNING_SERVER=full_server,
+                PRELIMINARY_SIGNING_SERVER=prelim_server):
+            sign_addons(addon_ids, force=options['force'])
```

Comment:
Just double checking that this calls the Python function directly and doesn't use the task `delay` to use celery. I imagine this is fine since it's a management command and you may want to run in the command and see output, etc.